QueryBuilder/Parsing/Expression.php

```diff
@@ -33,6 +34,12 @@ public function __construct($expression, $alias = null)
             }
         }
 
+        if (is_array($expression) && isset($expression['hidden']) && $expression['hidden']) {
+            $this->hidden = true;
```

Comment:
Can be simplified down to:

`$this->hidden = is_array($expression) && isset($expression['hidden']) && $expression['hidden']`

Reply:
fthank, fixe